step.yml

```diff
@@ -2,31 +2,52 @@ title: "Deploy to Bitrise.io - Apps, Logs, Artifacts"
+  1. Set the value for the **Deploy directory or file path** required input. It can be a path to a directory or a file. If you provide a directory, everything in that directory gets uploaded recursively. 
+  If you provide only a file, then only that file gets uploaded. The default value is the `$BITRISE_DEPLOY_DIR` Env Var which is exposed by the Bitrise CLI.
+  2. Set the value of the **Notify: User Roles** input. It sends an email with the [public install URL](https://devcenter.bitrise.io/deploy/bitrise-app-deployment/) to those Bitrise users whose roles are included in this field. 
+  The default value is `everyone`. If you wish to notify based on user roles, add one or more roles and separate them with commas, for example, `developers`, `admins`. If you don't want to notify anyone, set the input to `none`.
+  3. Set the **Notify: Emails** sensitive input. It sends the public install URL in an email to the email addresses provided here. If you’re adding multiple email address, make sure to separate them with commas. 
+  The recipients do not have to be in your Bitrise team. Please note that if the email address is associated with a Bitrise account, the user must be [watching](https://devcenter.bitrise.io/builds/configuring-notifications/#watching-an-app) the app.
+  4. The **Enable public page for the App?** required input is set to `true` by default. It creates a long and random URL which can be shared with those who do not have a Bitrise account. 
+  If you set this input to `false`, the **Notify: Emails** input will be ignored and the **Notify: User Roles** will receive the build URL instead of the public install URL.
+  5. With the **Compress the artifacts into one file?** required input set to `true`, you can compress the artifacts found in the Deploy directory into a single file.
+  You can specify a custom name for the zip file with the `zip_name` option. Otherwise the default Deploy directory will be used. 
+  If the **Compress the artifacts into one file?** is set to `false`, the artifacts in the Deploy directory will be deployed separately.
+  6. With the **Format for the BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP output** required input field, you can customize the output format of the public install page’s multiple artifact URLs so that the next Step can render the output (for example, our **Send a Slack message** Step). 
+  Provide a language template description using [https://golang.org/pkg/text/template]([https://golang.org/pkg/text/template) so that the **Deploy to Bitrise.io** Step can build the required custom output.
+  7. With the **Format for the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output** required input, you can customize the output format of the `BITRISE_PERMANENT_DOWNLOAD_URL_MAP` so that the next Step can render the output. 
+  This input creates a permanent URL containing all the artifacts the build has generated.
```

Comment:
`This input creates a permanent URL containing all the artifacts the build has generated.`

The input will be used to generate the related output in the specified format. The output contains multiple permanent urls for the multiple artifacts.

Can't we reuse the same content that is written for the public format?

Reply:
We can,  but then I freaked out since I did not mention the permanent side of things which is quite important with this input so I added that part. I like this solution: The input will be used to generate the related output in the specified format. The output contains multiple permanent urls for the multiple artifacts.